src/main/java/io/confluent/connect/elasticsearch/ElasticsearchSinkConnectorConfig.java

```diff
@@ -51,6 +53,12 @@ protected static ConfigDef baseConfigDef() {
           .define(CONNECTION_URL_CONFIG, Type.LIST, Importance.HIGH,
                   "List of Elasticsearch HTTP connection URLs e.g. ``http://eshost1:9200,http://eshost2:9200``.",
                   group, ++order, Width.LONG, "Connection URLs")
+          .define(CONNECTION_USERNAME, Type.STRING, Importance.LOW,
+                  "The username to use for HTTP Basic Authentication.",
+                  group, ++order, Width.LONG, "Connection Username")
```

Comment:
Both of these fields should probably be `Width.MEDIUM` -- usernames and passwords aren't all that long compared to, e.g., the list of connection URLs.